Assets/Scripts/UI/PickupPup/DogWorld/DogWorldSlot.cs

```diff
@@ -50,5 +50,15 @@ void handleScoutingDone()
     {
         this.Show();
     }
-        
+
+    /* BP Unsure if this is the preferred spot for this, but we apparently
+     * don't want any click noises for the dogs in the scene (yard/living room) */
+    void Start()
```

Comment:
On of the features of using `MonoBehaviourExtended` (which should be the superclass anytime you want to create a `MonoBehaviour` sciprt) https://github.com/PickupPup/Pickup-Pup/blob/master/Assets/Scripts/Core/TemplateClasses/MonoBehaviourExtended.cs is that we put all of the `Start` and `Awake` calls inside `fetchReferences` and `setReferences` respectively. 

Otherwise, this overwrites the functionality in superclasses, versus being able to call parent functionality using `base.methodName()` syntax